app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
```

Comment:
Your error message is not descriptive enough. Try to make it more specific. For example, you can change it to: `raise NotVaccinatedError('The visitor is not vaccinated.')`